docs/source/changelog.rst

```diff
@@ -21,6 +21,18 @@ We strongly recommend that you upgrade pip to version 9+ of pip before upgrading
     Use ``pip install pip --upgrade`` to upgrade pip. Check pip version with
     ``pip --version``.
 
+
+.. _release-6.0.0:
+
+6.0.0
+-----
+
+- add ``?no_track_activity=1`` argument to allow API requests
+  to not be registered as activity (e.g. API calls by external activity monitors).
+- Kernels shutting down due to an idle timeout is no longer considered
+  an activity-updating event.
```

Comment:
Thanks - can I ask you to make it explicit that the `shutdown_no_activity_timeout` therefore starts counting from the last time a kernel is used, rather than starting from when the last kernel shuts down.